packages/royalties/contracts/RoyaltyDistributer.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC2981Upgradeable} from "@openzeppelin/contracts-upgradeable/interfaces/IERC2981Upgradeable.sol";
+import {IRoyaltyManager} from "./interfaces/IRoyaltyManager.sol";
+
+contract RoyaltyDistributer is IERC2981Upgradeable {
+    uint16 internal constant TOTAL_BASIS_POINTS = 10000;
+    IRoyaltyManager public royaltyManager;
+
+    function __RoyaltyDistributer_init(address _royaltyManager) internal {
+        royaltyManager = IRoyaltyManager(_royaltyManager);
+    }
+
+    /// @notice Returns how much royalty is owed and to whom based on ERC2981
+    /// @param _tokenId of catalyst for which the royalty is distributed
+    /// @param _salePrice the price of catalyst on which the royalty is calculated
+    /// @return receiver the receiver of royalty
+    /// @return royaltyAmount the amount of royalty
+    /* solhint-disable-next-line no-unused-vars*/
```

Comment:
why no-unused-vars? Why have tokenId if it's never used?

Reply:
Used in the royaltyInfo function from EIP 2981. I could not remove it.

Reply:
Ok please add a dev comment